deployment/docker/docker-compose.yml

```diff
@@ -35,15 +35,22 @@ services:
       # SCRUMLR_SERVER_REDIS_HOST: "${REDIS_HOST}"
       # SCRUMLR_SERVER_REDIS_USERNAME: "${REDIS_USERNAME}"
       # SCRUMLR_SERVER_REDIS_PASSWORD: "${REDIS_PASSWORD}"
-    ports:
-      - "8080:8080"
```

Comment:
good catch! But what happens if somebody wants to do API requests directly against the backend or is this not a thing?